pkg/speculator/speculator.go

```diff
@@ -28,14 +28,23 @@ import (
 
 type SpecKey string
 
+type Config struct {
```

Comment:
Maybe SpeculatorConfig is more clear

Reply:
it's in the speculator package (looks o.k to me)

Reply:
The IDE warn on `Name starts with package name`.
It is clear that the Config is for the `Speculator` since it is under the `speculator` pkg.